Netmiko (#26716)
demisto/content: Packs/Netmiko/Integrations/Netmiko/Netmiko_description.md

## Netmiko SSH module integration

> This integration provides ssh-based access to network devices, servers, and other appliances that support this method of configuration. For a complete list of supported platforms, please visit the below URL:

Reviewer: ```suggestion
This integration provides ssh-based access to network devices, servers, and other appliances that support this method of configuration. For a complete list of supported platforms, visit the following URL:
```

[Netmiko Platforms.md on Github](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md)

## Configuration Parameters

>  - ***Name*** - Integration instance name

Reviewer: ```suggestion
 - *Name* - Integration instance name.
```

>  - ***Platform*** - The Netmiko-specific platform name

Reviewer: ```suggestion
 - *Platform* - The Netmiko-specific platform name.
```

>  - ***Hostname*** - The IP address, Hostname, or FQDN to connect to over SSH

Reviewer: ```suggestion
 - *Hostname* - The IP address, hostname, or FQDN to connect to over SSH.
```

>  - ***Port*** - The port to use for the SSH connection

Reviewer: ```suggestion
 - *Port* - The port to use for the SSH connection.
```

>  - ***Credentials*** - The credentials should be the same as the Tanium client.

Reviewer: ```suggestion
 - *Credentials* - The credentials should be the same as the Tanium client.
```
